db/data_migrate/20220414233400_add_isa_to_list.rb

```diff
@@ -0,0 +1,13 @@
+class AddCpToList < ActiveRecord::Migration[7.0]
+  def change
+    Organization.create!(
+      name: "ISA / Accelaero",
+      pegged: true,
+      great_place_to_work: false,
+      pegged_currency: 'AED',
+      remarks: 'Salary will be fully pegged. EPF, ETF will be calculated on the pegged salary. Will be applicable from April',
+      pegging_value: '55 AED',
```

Comment:
`pegging_value` is an integer https://github.com/dilumn/peggedplacetowork/blob/master/db/schema.rb#L43, so this need to be `pegging_value: 55,`

Reply:
PR updated. thanks